### ISSUE TITLE
Support for Norwegian Nynorsk

### ISSUE THREAD

**Issue by emilmuller** [explosion/spaCy#4417]
It would be great to also have a model for Norwegian Nynorsk (in addition to Bokmål). If I'm not mistaken, the Bokmål model is based on data from the University of Oslo. The University of Oslo usually releases Bokmål data together with Nynorsk data, so I'm guessing the exact same databases can be obtained as was used in training the Bokmål model.

**Comment by adrianeboyd** [comment]
Hi, I think the reason that there's only a Bokmål model is that the NER annotations are only available for Bokmål for that corpus (from https://github.com/ltgoslo/norne). spacy typically only maintains models for languages where there is also NER data available, although the NER annotation does not necessarily need to be for the same texts as the tagger or parser. If you know of NER data for Norwegian Nynorsk that is available under an acceptable license, let us know!

If you just need a tagger or a parser, I think you should be able to convert the UD / CONLL-U versions of the treebank using `spacy convert` and train a model with `spacy train` without too much trouble, but we haven't tried it ourselves. I would expect the `Language` and tokenizer settings stored in `spacy/lang` to be fairly similar, but I don't know enough about the detailed differences myself.

**Comment by emilmuller** [comment]
Ah, I see :) I visited that repo, but didn't catch the part regarding the NER missing from Nynorsk.

**Comment by ines** [comment]
Merging this with ##3056 for now 🙂 But we'd be happy to revisit this if we can get NER data for Nynorsk.

**Comment by nilutz** [comment]
> only a Bokmål model is that the NER annotations are only available for Bokmål for that corpus (from https://github.com/ltgoslo/norne).

Actually the corpus contains NER annotations for both languages. Currently we are using that corpus for training a new model with spacy.

See line 3, from the [dev.conllu](https://raw.githubusercontent.com/ltgoslo/norne/master/ud/nno/no_nynorsk-ud-dev.conllu) file

```# sent_id =  014303
# text = - Men Hellas maktar ikkje ti år med nedgang.
1	-	$-	PUNCT	_	_	4	punct	_	name=O
2	Men	men	CCONJ	_	_	4	cc	_	name=O
3	Hellas	Hellas	PROPN	_	_	4	nsubj	_	name=B-GPE_LOC
4	maktar	makte	VERB	_	Mood=Ind|Tense=Pres|VerbForm=Fin	0	root	_	name=O
5	ikkje	ikkje	PART	_	Polarity=Neg	4	advmod	_	name=O
6	ti	ti	NUM	_	Number=Plur|NumType=Card	7	nummod	_	name=O
7	år	år	NOUN	_	Definite=Ind|Gender=Neut|Number=Plur	4	obj	_	name=O
8	med	med	ADP	_	_	9	case	_	name=O
9	nedgang	nedgang	NOUN	_	Definite=Ind|Gender=Masc|Number=Sing	7	nmod	_	SpaceAfter=No|name=O
10	.	$.	PUNCT	_	_	4	punct	_	name=O```

Using this data and training the model is very similiar to the Bokmål version.

**Comment by adrianeboyd** [comment]
@nilutz That's good to know, thanks! We can see if we can update the conllu conversion script a bit to handle both `SpaceAfter` and `name=` so that it's easier to train models with this data.

**Comment by nilutz** [comment]
norne provides a script in scripts/ud2spacy.py. The Docs state: 

> The 10th MISC column will be converted from SpaceAfter=No|name=B-GPE_ORG to only include B-GPE_ORG

Maybe thats what you are looking for.

**Comment by adrianeboyd** [comment]
I'd kind of like to support the `SpaceAfter` in general, so we have raw texts for the tokenizer to work with. I wonder if there is a more standard name used for NER annotation than `name=`, though...

**Comment by svlandeg** [comment]
Looks like this got addressed by PR #4715 👍
